lib/zoom/actions.rb

```diff
@@ -15,8 +15,8 @@ def self.parse_path(path, path_keys, params)
       parsed_path
     end
 
-    def self.make_request(client, method, parsed_path, params, base_uri, url_encoded: false)
-      request_options = { headers: url_encoded ? client.form_url_encoded_request_headers : client.request_headers }
+    def self.make_request(client, method, parsed_path, params, base_uri, headers = nil)
```

Comment:
Five positional parameters is a bit much. I would suggest passing in a new object, a hash, or using keyword args.